studio/components/interfaces/Reports/ReportWidget.tsx

```diff
@@ -77,13 +78,24 @@ const ReportWidget: React.FC<ReportWidgetProps> = (props) => {
                   icon={<IconExternalLink strokeWidth={1.5} />}
```

Comment:
i do think that we shouldn't tie this button too much to opening it in editor/explorer, as it reduces possible use cases (like opening in a main report, or opening in usage report). Can refactor in the future.